src/content/docs/iast/troubleshoot-iast.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Troubleshooting for interactive applications security testing (IAST)
+tags:
+  - IAST
+  - Exploitable vulnerabilities
+  - Troubleshooting
+metaDescription: Use Vulnerability Management to find exploitable vulnerabilities in your application.
+---
+
+## I don't see my application in New Relic
+
+Confirm that the application has started and then review the application's logs for further information.
```

Comment:
How do I confirm that? Where do I find the application's logs?

Reply:
This is your own application logs. That will depend on how your application stores logs. I'll make it more obvious that that's what you need to check.